quinn/tests/post_quantum.rs

```diff
@@ -0,0 +1,105 @@
+#![cfg(feature = "rustls-aws-lc-rs")]
+use proto::crypto::rustls::HandshakeData;
+use quinn::{
+    crypto::rustls::{QuicClientConfig, QuicServerConfig},
+    Endpoint,
+};
+use rustls::{
+    pki_types::{CertificateDer, PrivatePkcs8KeyDer},
+    NamedGroup,
+};
+
+use std::{
+    error::Error,
+    net::{Ipv4Addr, SocketAddr},
+    sync::Arc,
+};
+
+#[tokio::test]
+async fn post_quantum_key_exchange() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    rustls_post_quantum::provider().install_default().unwrap();
+    let server_addr = (Ipv4Addr::LOCALHOST, 8080).into();
+
+    let (endpoint, server_cert) = make_server_endpoint(server_addr)?;
+    // accept a single connection
+    tokio::spawn(async move {
+        let incoming_conn = endpoint.accept().await.unwrap();
+        let conn = incoming_conn.await.unwrap();
+        println!(
+            "[server] connection accepted: addr={}",
+            conn.remote_address()
+        );
```

Comment:
Use tracing events instead of `println!()` (and set up the usual subscriber).

Reply:
I've done this; I've also added the `with_test_writer` to the `subscriber` because I noticed otherwise the log weren't printed.